include/solvers/simulation_parameters.h

```diff
@@ -135,6 +135,73 @@ class SimulationParameters
     particlesParameters.parse_parameters(prm);
     void_fraction->parse_parameters(prm);
     multiphysics.parse_parameters(prm);
+
+    // Create output_folder if does not exist
+    //    if (Utilities::MPI::this_mpi_process(this->mpi_communicator) == 0)
+    //      {
+    //        std::filesystem::create_directory(
+    //          simulation_control.output_folder.c_str());
+    //        //        _mkdir(simulation_control.output_folder.c_str());
+    //        std::cout << "Output folder created: "
+    //                  << simulation_control.output_folder << std::endl;
+    //      }
+
+    //    if (simulation_control.make_folder)
+    //      {
+    //        if (mkdir(simulation_control.output_folder.c_str(), 0777) == -1)
+    //          {
+    //            if (errno != EEXIST)
+    //              {
+    //                // if error other than "already exists"
+    //                std::cerr << "Could not create Output folder:  "
+    //                          << strerror(errno) << std::endl;
+    //              }
+    //          }
+    //        else
+    //          std::cout << "Output folder created: "
+    //                    << simulation_control.output_folder << std::endl;
+    //      }
```

Comment:
What to do about this block?

Reply:
These are what worked on my computer + a test to get it CI-compatible, but we might need to use another C++ version as discussed together. It will be cleaner to remove it for the PR, I will keep it on my local branch to remember that I want to put it in a upcoming PR :)